README.md

```diff
@@ -21,3 +21,19 @@ Scripts should be "chmod +x" executable in order to run as a cron job successful
 ## SES usage within alma-scripts
 * Emails from the SES service must come from `noreply@libraries.mit.edu` for this app 
 
+### Using Moto for local SSM testing
+
+It is often desireable to use [Moto](https://github.com/spulec/moto) for local development using the [Standalone Server Mode(https://github.com/spulec/moto#stand-alone-server-mode)] rather than using the actual SSM Parameter Store.
+
+To use, start moto running sqs in standalone mode with `pipenv run moto_server`, then:
+
+- add `SSM_ENDPOINT_URL='http://localhost:5000'` and `SSM_PATH="test/example/"` to your `.env` file
```

Comment:
I would set the SSM_PATH for local development to something different than the path we use for tests. Maybe "/dev/"?